luthier/config/src/dimensions.lua

```diff
@@ -121,6 +122,15 @@ local searchProviders = dimensionUtils.searchProviders
 local keyValueStores = dimensionUtils.keyValueStores
 
 return {
+    testDimension = {
+        longName = "a longName for testing",
+        description = "a description for testing",
+        fields = FIELDSETS.test,
+        category = "a category for testing",
+        type = "TestDimensionFactory",  -- type
+        searchProvider=searchProviders.noop,
+        keyValueStore=searchProviders.noop
```

Comment:
Should be `keyValueStores.noop`

Reply:
Fixed.

Reply:
Actually there is no keyValueStore.noop
I will use .memory instead. This prompts another change to make because all the other dimensions are wrongly populated previously.